Projects/python_patterns/menspants.py

```diff
@@ -0,0 +1,11 @@
+rows = 14
+print("*" * rows, end="\n")
+i = (rows // 2) - 1
+j = 2
+while i != 0:
+    while j <= (rows - 2):
+        print("*" * i, end="")
+        print("_" * j, end="")
```

Comment:
Men's pant's legs aren't connected, right?
replace that underscore with white space